README.md

```diff
@@ -1035,17 +1096,17 @@ Options for the [OpenID Connect](http://developer.okta.com/docs/api/resources/oi
     idpDisplay: 'PRIMARY'
     ```
 
-- **oAuthTimeout:** Timeout for OIDC authentication flow requests, in milliseconds. If the authentication flow takes longer than this timeout value, an error will be thrown and the flow will be cancelled. Defaults to `12000`.
+- **oAuthTimeout:** Timeout for [OIDC][] authentication flow requests, in milliseconds. If the authentication flow takes longer than this timeout value, an error will be thrown and the flow will be cancelled. Defaults to `12000`.
 
     ```javascript
     oAuthTimeout: 300000 // 5 minutes
     ```
 
 - **authParams:** An object containing configuration which is passed directly to the internal `authClient`. Selected options are described below. See the full set of [Configuration options](https://github.com/okta/okta-auth-js#configuration-options)
 
-- **authParams.pkce:** Set to `false` to disable PKCE flow
+- **authParams.pkce:** Set to `false` to disable [PKCE][] flow
```

Comment:
Should we be more explicit that disabling this reverts to implicit flow?

Reply:
it could also be authorization_code flow with a web application